packages/protobuf/scripts/protobuf-definitions.ts

```diff
@@ -0,0 +1,156 @@
+/* eslint-disable no-console */
+import fs from 'fs';
+import path from 'path';
+import * as protobuf from 'protobufjs';
+
+// protobuf.ReflectionObject to JSON
+type Definition = {
+    reserved?: unknown[];
+    options?: Record<string, unknown>;
+    valuesOptions?: Record<string, unknown>;
+    rule?: string;
+    type?: string;
+    nested?: Record<string, Definition>;
+    fields?: Record<string, Definition>;
+};
+
+const modifyDefinitionsJSON = (def: Definition, root: protobuf.Root) => {
+    // remove "reserved" fields
+    delete def.reserved;
+    delete def.valuesOptions;
+
+    // remove unused "options"
+    const ignoreOptions = ['(experimental_field)', '(experimental_message)', '(has_bitcoin_only_values)', 'deprecated'];
+
+    if (def.options) {
+        const options = Object.keys(def.options);
+        if (def.type && options.includes('packed')) {
+            const obj = root.lookup(def.type);
+            if (obj) {
+                try {
+                    // remove "packed" option from custom types
```

Comment:
cc @mroz22 - compatibility with previous json, otherwise it will add option.packed = false